package.json

```diff
@@ -4,7 +4,7 @@
 	"description": "Cookie Analysis Tool and CLI for analysis and understanding of cookie usage on web pages.",
 	"scripts": {
 		"build-storybook": "storybook build",
-		"build:all": "run-p build:tsc-packages build:cli-dashboard webpack:build:cli webpack:build:ext",
+		"build:all": "npm run build:tsc-packages && npm run  build:cli-dashboard && npm run webpack:build:cli && npm run webpack:build:ext",
```

Comment:
What issue did we have with `run-p` running all commands in parallel?

Reply:
`run-p` sometimes will run in parallel, meaning it will build some tsc packages after webpack is done building. This might break the build.